docs/MSPX.md

```diff
@@ -24,28 +26,87 @@ MspX is enabled by this setting in the receiver:
 
 For enabling MSP-RC set:
 
-- "Rx Snd RcChannel" = "rc override" or "rc channels"
+- "Rx Snd RcChannel" = "rc override" or "rc channels" (both have the same functionality in mspX mode)
 
 With MSP-RC enabled, SET_RAW_RC messages are sent to the flight controller. In the flight controller configure the receiver to the MSP protocol. This allows one to avoid the extra wire for CRSF or SBus. MspX needs to be enabled.
 
+***Notes***:
+RC Link Statistics are only send to the Flight controller via MSP, if "rc override" or "rc channels" is enabled. 
```

Comment:
Capital inconsistencies:

'RC link statistics are only sent to the flight controller via MSP if "rc override" or "rc channels" is selected.'